pkg/relayer/relayminer.go

```diff
@@ -134,3 +136,48 @@ func (rel *relayMiner) ServePprof(ctx context.Context, addr string) error {
 
 	return nil
 }
+
+// ServePing exposes ping HTTP server to check the reachability between the
+// relay miner and its dependencies (Ex: relay server and their respective
+// backend URLs).
+func (rel *relayMiner) ServePing(ctx context.Context, network, addr string) error {
+	ln, err := net.Listen(network, addr)
+	if err != nil {
+		return err
+	}
+
+	// Start a long-lived goroutine that starts an HTTP server responding to
+	// ping requests. A single ping request on the relay server broadcasts a
+	// ping to all backing services/data nodes.
```

Comment:
```suggestion
	// StartPingServer launches a goroutine that:
	// - Creates a long-running HTTP server
	// - Handles ping requests by broadcasting health checks to all backing services
	// - Tests connectivity to all configured data nodes
```

Reply:
`StartPingServer` feels like comment for a method and function but there is none defined. Could lead to confusion? wdyt?
I've brought some modifications based on your comment: 218006f7b0989c29d5ef7795178abb7849140131